qemu/tests/cfg/sev_basic_config.cfg

```diff
@@ -0,0 +1,15 @@
+- sev_basic_config:
+    type = sev_basic_config
+    kill_vm = yes
+    login_timeout = 240
+    image_snapshot = yes
+    mem = 4096
+    vm_secure_guest_type = sev
+    vm_sev_reduced_phys_bits = 1
+    vm_sev_cbitpos = 51
+    sev_enable_check = "journalctl|grep -i {policy_keyword}"
```

Comment:
do we want to add -w option for a strict single word check?

Reply:
Thanks, update with -w.